File: NewsApp1/app/src/main/java/com/example/myapplication/model/Article.kt
package com.example.myapplication.model

import kotlinx.serialization.Serializable

@Serializable
data class Article(
    val id: Int,
    val title: String,
    val content: String,
    val time: String,
    val isHotNews: Boolean,
    val categoryId: Int,
    val imageName: String?,
)

File: WebAPI/nhom 13/data/Article.cs
namespace nhom_13.data
{
    public class Article
    {
        public int Id { get; set; }

        public string Title { get; set; }

        public string Content { get; set; }

        public DateTime Time { get; set; }

        public Boolean IsHotNews { get; set; }

        public int CategoryId { get; set; }
        public Category Category { get; set; }

        public Image? Image { get; set; }
    }
}
